elastic/kibana
# [Lens] Support index pattern runtime fields in existence and field stats API

Fixes https://github.com/elastic/kibana/issues/90175 This PR reads the index pattern and includes index pattern level runtime fields for the requests of field existence and field stats API

To test:
1. Download and import index pattern including runtime field definition [index-pattern-so.ndjson.txt](https://github.com/elastic/kibana/files/5919154/index-pattern-so.ndjson.txt) 
2. Go to Lens and select imported index pattern
3. Runtime field `what_a_fielding` should work correctly

To simplify a little, this PR also changes the field stats API to include the index pattern id and field name instead of the index pattern title and field definition (as these things can be looked up on the server side). I don't have a strong opinion on this, but it looks like a simpler API to me (which probably wasn't possible in earlier versions because index patterns were not available on the server)
